### PR TITLE
docs: Add contributing file

### PR DESCRIPTION
**Pull request by jletey**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Brilliant. Now you can use Rust.
+
+## Running the API
+
+```sh
+$ cargo run # or ``cargo watch -x run``
+```
+
+Now visit `http://127.0.0.1:8080/graphiql`
```
Comment (tbaumer22) [inline CONTRIBUTING.md:106]: I’m reviewing this from a phone, but I wanted to verify that “graphiql” is the correct route instead of “graphql”?

Comment (divy-work) [reply CONTRIBUTING.md:106]: Yes it is, `graphiql` is the UI for graphql

**Comment by tbaumer22** [comment]
Feel free to merge then! Thanks for looking it over.

**Comment by jletey** [comment]
Thanks for the review @tbaumer22 and @divy-work. Looks like my changes break the linter tho 🤔

**Pull request merged**